integration_tests/base_routes.py

```diff
@@ -270,12 +270,32 @@ def file_download_sync():
 
 
 @app.get("/file_download_async")
-def file_download_async():
+async def file_download_async():
     current_file_path = pathlib.Path(__file__).parent.resolve()
     file_path = os.path.join(current_file_path, "downloads", "test.txt")
     return serve_file(file_path)
 
 
+@app.get("/binary_output_sync")
+def binary_output(request):
+    return b"OK"
+    # response = Response(
+    #     status_code=200,
+    #     headers={"Content-Type": "application/octet-stream"},
+    #     body=b"OK",
+    # )
+
+
+@app.get("/binary_output_async")
+async def binary_output(request):
+    return b"OK"
+    # response = Response(
+    #     status_code=200,
+    #     headers={"Content-Type": "application/octet-stream"},
+    #     body=b"OK",
+    # )
```

Comment:
Can you make two different test cases for this? That way we will be increasing the code coverage as we will still be testing the `Response` constructor type syntax and also the `b"` type syntax